Detectors/TOF/reconstruction/src/Decoder.cxx

```diff
@@ -13,58 +13,115 @@
 #include <chrono>
 #include <algorithm>
 #include "CommonConstants/LHCConstants.h"
-#define VERBOSE
+#include "TString.h"
+#include "FairLogger.h"
+//#define VERBOSE
 
 namespace o2
 {
 namespace tof
 {
 namespace compressed
 {
+Decoder::Decoder()
+{
+  for (int i = 0; i < NCRU; i++) {
+    mIntegratedBytes[i] = 0.;
+    mBuffer[i] = nullptr;
+    mCruIn[i] = false;
+  }
+}
 
 bool Decoder::open(std::string name)
```

Comment:
here also ``const std::string``. Could you please grep all your code for that?

Reply:
done